src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/SimpleNovelAdjacencyInterpreter.java

```diff
@@ -0,0 +1,214 @@
+        final Broadcast<ReferenceMultiSource> referenceBroadcast = svDiscoveryInputData.referenceBroadcast;
+
+        final JavaRDD<SimpleNovelAdjacency> simpleNovelAdjacencies =
+                getSimpleNovelAdjacencyJavaRDD(assemblyContigs, svDiscoveryInputData);
+
+        return simpleNovelAdjacencies
+                        .mapToPair(simpleNovelAdjacency ->
+                                new Tuple2<>(simpleNovelAdjacency,
+                                        inferSimpleOrBNDTypesFromNovelAdjacency(simpleNovelAdjacency,
+                                                referenceBroadcast.getValue(), referenceSequenceDictionaryBroadcast.getValue())));
+    }
+
+    private JavaRDD<SimpleNovelAdjacency> getSimpleNovelAdjacencyJavaRDD(final JavaRDD<AssemblyContigWithFineTunedAlignments> assemblyContigs,
+                                                                         final SvDiscoveryInputData svDiscoveryInputData) {
+        final String sampleId = svDiscoveryInputData.sampleId;
+        final Logger toolLogger = svDiscoveryInputData.toolLogger;
+        final Broadcast<SAMSequenceDictionary> referenceSequenceDictionaryBroadcast = svDiscoveryInputData.referenceSequenceDictionaryBroadcast;
+        final Broadcast<ReferenceMultiSource> referenceBroadcast = svDiscoveryInputData.referenceBroadcast;
+        final DiscoverVariantsFromContigsAlignmentsSparkArgumentCollection discoverStageArgs = svDiscoveryInputData.discoverStageArgs;
+        final List<SVInterval> assembledIntervals = svDiscoveryInputData.assembledIntervals;
+        final Broadcast<SVIntervalTree<VariantContext>> cnvCallsBroadcast = svDiscoveryInputData.cnvCallsBroadcast;
+
+        final JavaRDD<SimpleNovelAdjacency> simpleNovelAdjacencies =
+                assemblyContigs
+                        .filter(tig ->
+                                ChimericAlignment.splitPairStrongEnoughEvidenceForCA(tig.getSourceContig().alignmentIntervals.get(0),
+                                        tig.getSourceContig().alignmentIntervals.get(1),
+                                        MORE_RELAXED_ALIGNMENT_MIN_MQ, MORE_RELAXED_ALIGNMENT_MIN_LENGTH))
+                        .mapToPair(tig -> {
+                            final ChimericAlignment simpleChimera = ChimericAlignment.extractSimpleChimera(tig,
+                                    referenceSequenceDictionaryBroadcast.getValue());
+                            final byte[] contigSequence = tig.getSourceContig().contigSequence;
+                            final NovelAdjacencyReferenceLocations novelAdjacencyReferenceLocations =
+                                    new NovelAdjacencyReferenceLocations(simpleChimera, contigSequence,
+                                            referenceSequenceDictionaryBroadcast.getValue());
+                            final byte[] altHaplotypeSeq = null; // TODO: 1/21/18 force all subtypes of NovelAdjacencyReferenceLocations to extract alt haplotype
```

Comment:
Address TODO or add issue. What is the point of a final variable that is set to null?

Reply:
this is what I was talking about when we discussed providing extra alt haplotype sequence annotation in VCF. I have this in mind and it will be the topic of the next PR considering it will make your job considerably easier